.gitignore

```diff
@@ -274,3 +274,9 @@ config.ps1
 # VS generated files
 launchSettings.json
 
+# Snapcraft files
+.snapcraft
+*.snap
+parts/
+prime/
+stage/
```

Comment:
\<nit> Should keep a newline at the end to be nice.

Reply:
added